react/features/polls/reducer.ts

```diff
@@ -83,21 +83,37 @@ ReducerRegistry.register<IPollsState>('features/polls', (state = INITIAL_STATE,
         // if the poll exists, we update it with the incoming answer
         const newAnswers = state.polls[pollId].answers
             .map(_answer => {
+                const voters = _answer.voters as string[];
+
                 return {
                     name: _answer.name,
-                    voters: new Map(_answer.voters)
+                    voters: typeof voters?.length === 'number' ? [ ...voters ] : { ...voters }
```

Comment:
Can't we normalize this at this level? That is, if we get an object / map transfor it into an array?

Also, I feel like this needs a comment to make sure what's happening is properly understood.